src/Data/Graph/Typed.hs

```diff
@@ -0,0 +1,156 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module     : Algebra.Graph.KingLaunchbury
+-- Copyright  : (c) Anton Lorenzen 2016-2018
+-- License    : MIT (see the file LICENSE)
+-- Maintainer : andrey.mokhov@gmail.com
```

Comment:
@anfelor This looks a bit odd: you hold the copyright, but I'm the maintainer. I guess it makes sense to list both of us in both fields to avoid confusion. (For example, if someone finds a bug in this module, I hope you're OK to fix it!)

Reply:
Of course I am! I have also added you to the copyright, since most of the code was originally written by you.